README.md

```diff
@@ -84,6 +84,7 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
 
 # Where to start?
 * [Basics – Docker, Containers, Hypervisors, CoreOS](http://etherealmind.com/basics-docker-containers-hypervisors-coreos/)
+* [Dive Into Docker: The Complete Docker Course For Developers](https://www.youtube.com/watch?v=XeSD17YRijk&list=PL-v3vdeWVEsXT-u0JDQZnM90feU3NE3v8) (60:25) by [@nickjanetakis](https://twitter.com/nickjanetakis)
```

Comment:
It should be renamed to `Dive Into Docker: Getting Started` or something like that because what the content of the link provide is a list of high level videos about Docker concepts. It explains the architecture and comparison with VMs but I fail to see anything relevant for developers since the tutorial offers only one video about how to use docker and it's a hello world example. I understand that the paying tutorial contains more but the youtube playlist provided is really basic stuff. So the word `Complete` and `For Developers` are misleading in this case.